src/components/text-input-email.components.js

```diff
@@ -0,0 +1,23 @@
+import $ from 'jquery';
+import TextInput from './text-input.components';
+import InvalidEmail from '../shared/erros/text-input-email.erros';
+
+export default class EmailTextInput extends TextInput {
+    constructor(stringQuery, required = true, _textValid = false, disabled = false) {
+        super(stringQuery, required, _textValid, disabled);
+        this._textInput = $(`${stringQuery} input[type="email"]`);
+    }
+
+    isTextValid(email) {
```

Comment:
The component already has the value entered in the property, it does not need to pass a value to be validated

Reply:
fixed